yolov8/README.md

```diff
@@ -23,6 +23,12 @@ The tensorrt code is derived from [xiaocao-tian/yolov8_tensorrt](https://github.
 Currently, we support yolov8
 
 - For yolov8 , download .pt from [https://github.com/ultralytics/assets/releases](https://github.com/ultralytics/assets/releases), then follow how-to-run in current page.
+
+## Custom models
+
+- Use ultralytics<=8.2.103 for training.
```

Comment:
Add ultralytics <= 8.2.103 to Requirements, would be more clear.

Reply:
Sure, I'll fix it.